etc/calls/testcall.py

```diff
@@ -29,24 +30,26 @@
     #
     line = sys.stdin.readline()
     line = line.strip()
-    line = urllib.unquote(line.decode('utf-8'))
     # Remove Base64 encoding
-    mystr = base64.b64decode(line)
-    args = cPickle.loads(mystr)
+    pickledB64Arguments = base64.b64decode(urllib.parse.unquote(line))
```

Comment:
Change the name, it is confusing to use pickledB64Arguments when the value has nothing to do with B64